AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Configuration/AWSPinpointAnalyticsPluginConfiguration.swift

```diff
@@ -54,20 +54,20 @@ public struct AWSPinpointAnalyticsPluginConfiguration {
 
         let pluginConfiguration = try AWSPinpointPluginConfiguration(pinpointAnalyticsConfig)
 
-        let configOptions: AWSPinpointAnalyticsPlugin.Options
-        if let options {
-            configOptions = options
+        let configOptions: AWSPinpointAnalyticsPlugin.Options = if let options {
```

Comment:
Maybe it's just me, but this new syntax is way more confusing than the previous one.

Reply:
agreed, revert/remove conditional assignment rule.